### PR TITLE
Create frauddefence.md

### PR DESCRIPTION
**Pull request by mikefay**
This Fraud Defence page replaces our SSL documentation, and eventually all security features.

### CODE REVIEW

**Comment by aengusbates** [comment]
It probably makes more sense to keep the SSL doc separate from the Fraud Tools doc. It is a separate, niche discussion. 

This doc should cover CVV, AVS, and 3d Secure.  We can have a separate doc covering TLS.

See:
https://stripe.com/docs/fraud/guide/strategies
https://stripe.com/docs/security
https://articles.braintreepayments.com/guides/fraud-tools/overview

File: source/docs/guides/SSL.md [hunk 1]
```diff
@@ -1,64 +0,0 @@
----
-title: SSL Certificates
```
Comment (aengusbates) [inline source/docs/guides/SSL.md]: It probably makes more sense to keep the SSL doc separate from the Fraud Tools doc. It is a separate, niche discussion.

This doc should cover CVV, AVS, and 3d Secure. We can have a separate doc covering TLS.

See:
https://stripe.com/docs/fraud/guide/strategies
https://stripe.com/docs/security
https://articles.braintreepayments.com/guides/fraud-tools/overview

Comment (mikefay) [inline source/docs/guides/SSL.md]: That works for me. 3D Secure just needs a once over from Mike Fisher, then a code review. We can either inject the Fraud Defence elements and change the URL, or move 3D Secure over to Fraud Defence. I'll update this doc with the 3D Secure pieces after approval is complete.

**Comment by aengusbates** [comment]
Comprehensive security and fraud docs to be covered in separate pull. Closing pull.

**Pull request closed without merging**